publication-commons/src/test/java/no/unit/nva/publication/service/impl/TicketServiceTest.java

```diff
@@ -156,7 +157,7 @@ void shouldThrowErrorWhenDoiIsRequestedForIneligiblePublication(PublicationStatu
     void shouldCreatePublishingRequestForDraftPublication() throws ApiGatewayException {
         var publication = persistPublication(owner, DRAFT);
         var ticket = PublishingRequestCase.createOpeningCaseObject(publication);
-
+        ticket.setWorkflow(PublishingWorkflow.REGISTRATOR_PUBLISHES_METADATA_ONLY);
```

Comment:
Denne verdien kommer ikke med til TicketService

Reply:
Joda, hvis du ser på 
assertThat(persistedTicket, is(equalTo(ticket)));
så hadde den throwet en exception og testen hadde ikke kjørt.